cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/ArtifactStore.java

```diff
@@ -207,6 +212,9 @@
       RetryStrategies.retryOnConflict(20, 100)
     );
     this.impersonator = impersonator;
+    this.excludedRequirements =
+      Arrays.stream(configuration.getTrimmedStrings(Constants.Plugin.EXCLUDE_WITH_REQUIREMENTS))
```

Comment:
Use `new HashSet<>(cConf.getTrimmedStringCollection());`, which is simpler

Reply:
Done.